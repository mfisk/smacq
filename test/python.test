#! /bin/sh

TESTFILE=$TEST/0325@1112-snort.pcap
PYTHONPATH=$TEST; export PYTHONPATH

${BINDIR}smacqq -m <<EOF 
pcapfile $TESTFILE | python("pythontest.Dumper", "foobar") | print ts
EOF

exit 1
