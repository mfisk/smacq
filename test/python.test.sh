#! /bin/sh

TESTFILE=0325@1112-snort.pcap
PYTHONPATH=$TEST; export PYTHONPATH

$SMACQQ -f - <<EOF 
pcapfile $TESTFILE | python("pythontest.Dumper", "foobar") | print ts
EOF

exit 1
