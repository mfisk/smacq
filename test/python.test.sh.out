('init', ('foobar',))
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.210')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 93L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.210')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 97L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.210')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.210')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
2000-03-25T12:15:28.020237-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 126L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 191L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 346L)
2000-03-25T12:15:28.028445-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
2000-03-25T12:15:28.129182-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
2000-03-25T12:15:28.620389-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
2000-03-25T12:15:29.019047-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
2000-03-25T12:15:29.119403-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
2000-03-25T12:15:31.393278-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 126L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 191L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 256L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 158L)
2000-03-25T12:15:31.434037-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 194L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 117L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 246L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 99L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 154L)
2000-03-25T12:15:31.449118-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 256L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 146L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 154L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 194L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 117L)
2000-03-25T12:15:31.479025-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 222L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 99L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 154L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 194L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 117L)
2000-03-25T12:15:31.485619-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 218L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 206L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 194L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 236L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 218L)
2000-03-25T12:15:31.491501-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 198L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 194L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 188L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 190L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
2000-03-25T12:15:31.527459-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 270L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 194L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 188L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 190L)
2000-03-25T12:15:31.533792-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 270L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 194L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 188L)
2000-03-25T12:15:31.539540-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 190L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 270L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 194L)
2000-03-25T12:15:31.545646-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 188L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 190L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 270L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
2000-03-25T12:15:31.552164-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 186L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 62L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
2000-03-25T12:15:31.768626-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 126L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 210L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 99L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 97L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
2000-03-25T12:15:31.914011-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 93L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x11')
('len', 'uint32', <type 'long'>, 92L)
('srcip', 'ip', <type 'str'>, '192.168.0.200')
('dstip', 'ip', <type 'str'>, '192.168.0.54')
('ipprotocol', 'ipproto', <type 'str'>, '\x11')
('len', 'uint32', <type 'long'>, 307L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 346L)
2000-03-25T12:15:31.945492-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 346L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 328L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 328L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 330L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 330L)
2000-03-25T12:15:47.127418-07:00
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 328L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 328L)
('srcip', 'ip', <type 'str'>, '192.168.0.54')
('dstip', 'ip', <type 'str'>, '192.168.0.200')
('ipprotocol', 'ipproto', <type 'str'>, '\x06')
('len', 'uint32', <type 'long'>, 60L)
('del', 3)
